data/shaileshthakur1.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Shailesh Thakur",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Currently a 2nd year Undergrad,interested in project and research internship from abroad",
+  "avatar": "https://github.com/shaileshthakur1.png"
```

```suggestion
  "avatar": "https://github.com/shaileshthakur1.png"
```
```avatar``` property isnot required